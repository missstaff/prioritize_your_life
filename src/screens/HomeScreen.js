import { Text, View, useColorScheme } from "react-native";
import { useTheme } from "@react-navigation/native";
import { getFireApp } from "../../getFireApp";


const HomeScreen = () => {
    
    const firebase = getFireApp();
    const scheme = useColorScheme();
    let { colors } = useTheme();

    return (
        <View style={{ flex: 1, alignItems: "center", justifyContent: "center" }}>
            <Text style={{ color: colors.text }}>Home Screen</Text>

        </View>
    );
};

export default HomeScreen;