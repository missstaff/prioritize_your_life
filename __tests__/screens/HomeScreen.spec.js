import renderer from "react-test-renderer";
import HomeScreen from "../../src/screens/HomeScreen";


describe("HomeScreen", () => {
  it("HomeScreen should render correctly", () => {
    const tree = renderer.create(<HomeScreen />).toJSON()
    expect(tree).toMatchSnapshot()
  });
  afterEach(() => {
    jest.clearAllMocks();
  });
});