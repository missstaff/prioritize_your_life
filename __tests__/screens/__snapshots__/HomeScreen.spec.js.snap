// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen HomeScreen should render correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "color": "rgb(28, 28, 30)",
      }
    }
  >
    Home Screen
  </Text>
</View>
`;
